### DISCUSSION TITLE
"spawn EPERM" Error - Need help, Videos stopped downloading

### DISCUSSION THREAD

**Discussion by spookycat567** [aclap-dev/video-downloadhelper#229] [Help and Q&A]
### Your question

Hello, I need help in fixing my issue/
I recently tried downloading from a website where I was unable to download from, So i moved on and went back to downloading from the place where i usually download (which worked perfectly fine until now). After that, my Video Downloadhelper has not been downloading videos anymore. 

Now, when i try to download a video, i go through the same process and it appears as normal at first with the blue icon on top showing It's in progress but then in a matter of seconds it goes to a red icon. It reads "spawn EPERM (Details)".

I click on details and it says this:

Error: spawn EPERM
    at rs.receive (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:13157)
    at chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:42624

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: accepted
key (only 16 first characters): 9e573601b1ad41ae
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\1730j",
  "converterError": "spawn EPERM"
}

Note:

During this, I removed my Chrome extension and downloaded it again but it still does not work. (I have a license and when i downloaded it i made sure to register my license again)

In advance, Thank you!

**Comment by paulrouget** [comment]
You're not the first one to report this issue! I'm looking into this as soon as possible. Thank you for reporting the problem.

**Reply by spookycat567** [reply]
Thank you! I'll be here if you need anything and while we patiently wait!

**Reply by paulrouget** [reply]
> from the place where i usually download

Is this place in `C:\` or on a different disk?

**Reply by spookycat567** [reply]
Yes, correct, it’s on “ C : \ “


Note: sorry! I realized i didn't reply to ur comment, because I sent it through email.

**Comment by spookycat567** [comment]
Yes, correct, it’s on “ C : \ “
